src/tokens.ts

```diff
@@ -124,10 +124,11 @@ export interface IDebugger {
   clearBreakpoints(): Promise<void>;
 
   /**
-   * Request details for a variable.
-   * @param variable The variable for which to request details.
+   * Request variables for a given variable reference.
+   *
+   * @param variablesReference The variable reference to request.
    */
-  getVariableDetails(
+  inspectVariable(
```

Comment:
This is technically a breaking change, but it should be fine to make for `0.3.0` (still `0.x`) as it clarifies the interface.